### PR TITLE
Utest 0.7.6

### PR DESCRIPTION
**Pull request by sideeffffect**
/cc @lolgab

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -2,7 +2,7 @@ Global / onChangedBuildSource := ReloadOnSourceChanges
 
 inThisBuild(
   Seq(
-    organization := "com.github.lolgab",
+    organization := "dev.whaling",
```
Comment (lolgab) [inline build.sbt]: Since I'm maintaining the library @rwhaling proposed to use my groupid instead of `"dev.whaling"`
You should revert this change.

File: build.sbt [hunk 2]
```diff
@@ -19,8 +19,8 @@ val publishSettings = Seq(
   publishTo := sonatypePublishToBundle.value,
   scmInfo := Some(
     ScmInfo(
-      url("https://github.com/lolgab/scala-native-loop"),
-      "scm:git:git@github.com:lolgab/scala-native-loop.git"
+      url("https://github.com/scala-native/scala-native-loop"),
+      "scm:git:git@github.com:scala-native/scala-native-loop.git"
```
Comment (lolgab) [inline build.sbt:23]: Good catch! 👍

**Review by lolgab — CHANGES_REQUESTED** [review]
A little comment otherwise it's good to merge

File: build.sbt (hunk 1 above)
Comment (sideeffffect) [reply build.sbt]: :+1:

**Review by lolgab — APPROVED** [review]

**Comment by lolgab** [comment]
Thank you 😃

File: build.sbt (hunk 1 above)
Comment (sideeffffect) [reply build.sbt]: Btw in future we should probably migrate to `org.scala-native`, this repo is already hosted at `github.com/scala-native` anyway.
